src/app/sidenav/sidenav.component.html

```diff
@@ -1,22 +1,24 @@
 <mat-sidenav-container fullscreen [style.top.px]="utils.screenWidth <= 599 ? 56 : 64">
 
-  <mat-sidenav #sidenav [mode]="utils.isMobile ? 'over': 'side'" [opened]="!utils.isMobile" [style.width.px]="sidebarWidth">
+  <mat-sidenav #sidenav [mode]="utils.isMobile ? 'over': 'side'" [opened]="!utils.isMobile && sidebarState" [style.width.px]="sidebarWidth">
     <mat-nav-list>
       <a mat-list-item [routerLink]="['/create-community']">Create Community</a>
       <mat-divider></mat-divider>
       <mat-list-item *ngFor="let communityFav of communitiesFavourited; let j = index">
         <a mat-line *ngIf="communityFav.isFavourited" [routerLink]="['/']">
-           {{ communityFav.communityname }}
+           {{ communityFav.title }}
```

Comment:
This should be something like {{ communityFav.sidebar ? communityFav.sidebar : communityFav.title }} to allow for custom sidebar names (similarly for line 17)